spyder/plugins/explorer/widgets/fileassociations.py

```diff
@@ -147,7 +147,10 @@ def setup(self, applications=None):
         QApplication.setOverrideCursor(QCursor(Qt.WaitCursor))
         self.list.clear()
         if applications is None:
-            apps = get_installed_applications()
+            try:
```

Comment:
```suggestion
            # This is necessary to avoid an error on Windows for non-admin
            # accounts.
            # Fixes spyder-ide/spyder#20907
            try:
```